airflow/__init__.py

```diff
@@ -47,8 +47,6 @@
 
 login: Optional[Callable] = None
 
-PY36 = sys.version_info >= (3, 6)
-PY37 = sys.version_info >= (3, 7)
```

Comment:
I'm actually going to leave there here and in __all__ so we don't break backcompat.